.github/workflows/integration-test.yaml

```diff
@@ -72,48 +88,35 @@ jobs:
             exit 1
           fi
       
-      - name: Static Analyzer Reports Uploading
+      - name: Upload Static Analyzer Reports
         if: ${{ failure() && steps.analyzer-output.outcome == 'failure' }}
         uses: actions/upload-artifact@v2
         with:
-          name: static-analyzer-reports-test_macOS
+          name: static-analyzer-reports-${{ matrix.lane }}
           path: ./derived_data/**/report-*.html
-
-      - name: Run Examples Tests
-        working-directory: ./Examples/Tests
-        run: |
-          pod repo update
-          pod install
-          bundle exec fastlane scan -s Tests --output-directory "fastlane/test_output/examples/test_macOS"
       
-      - name: Xcodebuild Logs Artifact
+      - name: Upload Xcodebuild Logs
         if: always()
         uses: actions/upload-artifact@v2
         with:
           name: xcodebuild-logs
           path: ~/Library/Developer/Xcode/DerivedData/*/Logs
 
-      - name: Upload test output artifact
+      - name: Upload Test Output
         if: always()
         uses: actions/upload-artifact@v2
         with:
           name: test-output
           path: fastlane/test_output
 
-      - name: Upload test results to observability server
+      - name: Upload Test Results
         if: always()
         env:
           TEST_OBSERVABILITY_SERVER_AUTH_KEY: ${{ secrets.TEST_OBSERVABILITY_SERVER_AUTH_KEY }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        run: Scripts/upload_test_results.sh
-      
-      - name: Swift Package Manager - Installation Test
-        working-directory: ./
-        run:  |
-          echo 'Current Branch: ' $GITHUB_HEAD_REF
-          echo 'Current Revision (SHA):' $GITHUB_SHA
-          echo Current Path: $(pwd)
-          export PACKAGE_URL=file://$(pwd)
-          export PACKAGE_BRANCH_NAME=$GITHUB_HEAD_REF
-          export PACKAGE_REVISION=$GITHUB_SHA
-          swift test --package-path Examples/SPM -v
+          TEST_RESULTS_WEB_UI_URL: undefined
```

Comment:
Is this dance with `TEST_RESULTS_WEB_UI_URL` (it's not a common pattern for a script to write its output to an environment variable) and the extra job necessary? Can't we just wrap `upload_test_results.sh`’s `printf "Created request body..."` inside a `::group::Body of request`?

Reply:
https://github.com/ably/ably-cocoa/pull/1840/commits/6bd3ea8a8adf35b3290e4362ad9a4c9663cead07

Reply:
Nice, looks good